service-tests/json.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'json', title: 'User Defined JSON Source Data' });
```

Comment:
What do you think about making these URL paths something like `custom-json`? Could also be `user-json` or `json-uri` or similar.

It would make it clearer to someone looking at the URL what is happening. It would also be easier to search for in the codebase.

Reply:
Agreed just having `/json/` isnt the best name for it,
upto you which i should use though as i am not sure which would work best 😄 

some other options:
`json-source`
`source/json`
`dynamic/json`
`custom/json`
`badge/json`
`badge-json`
the bottom 2 could be useful to kind of build on top of the current static badge as you could have:
`badge/static`
`badge/xml`

Reply:
I like the idea of using `/badge`, since it groups together the custom badges, and agree the slashes look nice. How about `/badge/dynamic/json`?